network/f5/bigip_virtual_server.py

```diff
@@ -213,11 +218,40 @@ def vs_create(api,name,destination,port,pool):
 def vs_remove(api,name):
     api.LocalLB.VirtualServer.delete_virtual_server(virtual_servers = [name ])
 
+
+def get_rules(api,name):
+    return api.LocalLB.VirtualServer.get_rule(virtual_servers = [name])[0]
+
+
+def set_rules(api,name,rules_list):
+    updated=False
+    if rules_list is None:
+        return False
+    rules_list = list(enumerate(rules_list))
+    try:
+        current_rules=map(lambda x: (x['priority'], x['rule_name']), get_rules(api,name))
+        to_add_rules=[]
+        for i, x in rules_list:
+            if (i ,x) not in current_rules:
```

Comment:
rule_name under virtual server must be unique, but priority may not same,
so if use this way to check rule not in current_rules may be got wrong result.
then this rule will not be added under virtual server rule list.